SECURITY.md

```diff
@@ -18,6 +18,14 @@ As Git for Windows bundles more than just Git (such as Bash, OpenSSL, OpenSSH, G
 
 Every Git for Windows version is tagged using a name that starts with the Git version on which it is based, with the suffix `.windows.<patchlevel>` appended. For example, Git for Windows v2.17.1' source code is tagged as [`v2.17.1.windows.1`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.1) (the patch level is always at least 1, given that Git for Windows always has patches on top of Git). Likewise, Git for Windows v2.17.1(2)' source code is tagged as [`v2.17.1.windows.2`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.2).
 
+## Release Candidates (rc) and Snapshot versions ('nightlies')
+
+Git for Windows also releases versions that reflect the upstream release candidates, which contain the `-rc<n>` suffix to the regular git version, and before the 'windows' suffix. It should be noted that these rc versions sort after their formal release, so appear to be newer to the updater software.
```

Comment:
Maybe link to https://tinyurl.com/gitCal for the upstream release cycle? That would also give us an excuse to explain that Git for Windows tries to release new versions and prereleases as close to upstream Git, but they are not really tied together other than by convention.

As to -rc versions sorting incorrectly, this is arguably a bug in our installer, and I would love to see it fixed (hint: this is a really good opportunity to help).

Reply:
I've had a quick look at the issue with rc versions and regular releases, and I think I see the issue (well, 2 issues) and have an idea for a fix. But let me recap the issue symptoms real quick, so I don't we're on the same page as to what we're talking about:

If you're on an rc version, say `v2.23.0-rc1.windows.1` and the corresponding release version (`v2.23.0.windows.1`) gets released, `git update-git-for-windows` won't detect that as a newer version.

Did I understand that correctly? I usually skip the rcs, so I'm not quite sure.

Reply:
Ok, reading Philips comment in #2312 I might have missunderstood the issue.

>when upgrading from the rc build to the proper first release, you WILL get a dialog saying that it appears that you are down grading versions.

Reply:
Hi @rimrul 
I can't remember the exact sequence. I had one of the release candidates installed (I think I had both rc1 and rc2 as they came out and I was responding to the email announcements).

The formal release comes out and I get the next announcement (hence, I think, ahead of any daily check). I click some link, to bet to the announced release (here it is fuzzy).

I get the download from the appropriate page (I'm on Firefox) and it's stored into my downloads folder. View the downloads folder. 

Run the download/installer. Installer _warns_ that the release about to be installed is older than the current version - "Install anyway?" (or something like that). I install anyway.

That's the sequence, with some fuzzy 'didn't take notes at the time' inexactitudes.

Hope that helps.

Reply:
Yes, that helps. The version comparison just compares each number (one or more digits separated by one or more non-digit character) in both version numbers. Thus it gets confused by rc versions having more numbers than regular releases.

Reply:
Exactly!